src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int ONE = 1;
+    private static final int ZERO = 0;
+
+    public String getStatistic(String fromFileName, String toFileName) {
+        File inputFile = new File(fromFileName);
+        File outputFile = new File(toFileName);
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(inputFile))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                String[] split = value.split(",");
```

Comment:
Instead of using `substring()` or similar methods, it's recommended to use `split()` to separate the values. You are already using `split()` which is good, but consider defining a constant for the delimiter `,` to improve code readability.